lib/rspecq/worker.rb

```diff
@@ -185,5 +196,23 @@ def relative_path(job)
     def elapsed(since)
       Process.clock_gettime(Process::CLOCK_MONOTONIC) - since
     end
+
+    # Prints msg to standard output and emits an event to Sentry, if the
+    # SENTRY_DSN environment variable is set.
+    def log_event(msg, level, additional={})
+      puts msg
+
+      Raven.capture_message(msg, level: level, extra: {
+        build: @build_id,
+        worker: @worker_id,
+        queue: @queue.inspect,
```

Comment:
minor: This might grow out of hand for big queues :)
If no scenario where this proves helpful happens we might need to rethink about it.

Reply:
This doesn't contain all the jobs in the queue but merely the instance variables of the `@queue`:

```ruby
queue.inspect #<RSpecQ::Queue:0x0000556802b244f8 @build_id="641a3ea4", @worker_id="8b494bb4", @redis=#<Redis client v4.2.1 for 8b494bb4>>

Reply:
:+1: